tools/enclaveos-encrypted-fs/examples/setup-using-dsm.rs

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) Fortanix, Inc.
```

Comment:
Can you add this end to end test in github workflows, we can pass FORTANIX_API_KEY as the DSM_API_KEY variable needed to run this example.